### ISSUE TITLE
Total test time tally

### ISSUE THREAD

**Issue by OverLordGoldDragon** [spyder-ide/spyder-unittest#157]
Is there an option to show total time taken (sum) by all tests? If not, would be a helpful feature.

<img src="https://user-images.githubusercontent.com/16495490/82135557-245c8600-9815-11ea-96cf-bd80f0929687.png" width="150">

**Comment by jitseniesen** [comment]
No, there is no such option. I agree it could be useful, so hopefully somebody will implement it.

**Comment by StefRe** [comment]
In the meanwhile you could look at the bottom line of the test output (Unit testing menu | Show output):
`======================= 149 passed, 1 warning in 7.21s ========================`

**Comment by OverLordGoldDragon** [comment]
@StefRe Great, this'll do - though users might be as oblivious as I was per not checking the likely massive log dumps or just missing it.
